src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -41,15 +42,40 @@ import {
 import minimalData from './schema/minimal-test.json';
 import maximalData from './schema/maximal-test.json';
 
-import initialData from './initialData';
-
 import {
   SERVICE_CONNECTION_TYPES,
   PTSD_INCIDENT_ITERATION,
   PTSD_CHANGE_LABELS,
+  disabilityActionTypes,
 } from '../../all-claims/constants';
 
 describe('526 helpers', () => {
+  describe('filterServiceConnected', () => {
+    it('should filter non-service-connected disabililties', () => {
+      const disabilities = [
+        { decisionCode: SERVICE_CONNECTION_TYPES.notServiceConnected },
+        { decisionCode: SERVICE_CONNECTION_TYPES.serviceConnected },
+        { decisionCode: SERVICE_CONNECTION_TYPES.notServiceConnected },
+        { decisionCode: SERVICE_CONNECTION_TYPES.serviceConnected },
+      ];
+
+      const filteredDisabilities = filterServiceConnected(disabilities);
+      expect(filteredDisabilities.length).to.equal(2);
+      filteredDisabilities.forEach(d =>
+        expect(d.decisionCode).to.equal(
+          SERVICE_CONNECTION_TYPES.serviceConnected,
+        ),
+      );
+    });
+
+    it('should return an empty array when no disabilities provided', () => {
+      const disabilities = [];
+
+      const filteredDisabilities = filterServiceConnected(disabilities);
+      expect(Array.isArray(filteredDisabilities)).to.be.true;
```

Comment:
What do you think of changing this to `expect(filteredDisabilities).to.be.an('array').that.is.empty`?

(Apparently that's [a thing](https://www.chaijs.com/api/bdd/#method_a).)

Reply:
TIL, thanks!